### PR TITLE
feat: expose local enviroment with mocks

### PR DESCRIPTION
**Pull request by cbernat**
Link https://cdn.fromdoppler.com/doppler-webapp/demo-build1573/#/reports

### CODE REVIEW

File: build-n-publish.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ pkgVersion=${1:-"v0.0.0-build0"}
 cdnBaseUrl=${2:-"//cdn.fromdoppler.com/$pkgName"}
 pkgBuild=${4:-0}
 pkgCommitId=${5:-$(git rev-parse HEAD)}
-environments="int qa development production"
+environments="int qa development production local"
```
Comment (andresmoschini) [inline build-n-publish.sh]: What do you think about using `demo` in place of `local`?

Comment (cbernat) [reply build-n-publish.sh]: Sure, seems fine. Once this works.. 😄

File: src/index.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ const locale = availableLanguageOrDefault(navigator.language.toLowerCase().split
 
 // Only used in development environment, it does not affect production build
 const forcedServices =
-  process.env.NODE_ENV === 'development'
+  process.env.NODE_ENV === 'development' || process.env.REACT_APP_IS_DEMO_ENABLED
```
Comment (andresmoschini) [inline src/index.js]: 👍

File: src/index.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ const locale = availableLanguageOrDefault(navigator.language.toLowerCase().split
 
 // Only used in development environment, it does not affect production build
```
Comment (andresmoschini) [inline src/index.js]:
```suggestion
// Only used in development and demo environments, it does not affect production build
```

File: src/index.js [hunk 4]
```diff
@@ -27,9 +27,9 @@ if (document.querySelector('body').setActive) {
 
 const locale = availableLanguageOrDefault(navigator.language.toLowerCase().split(/[_-]+/)[0]);
 
-// Only used in development environment, it does not affect production build
+// Only used in development and demo environment, it does not affect production build
```
Comment (andresmoschini) [inline src/index.js]:
```suggestion
// Only used in development and demo environments, it does not affect production build
```
😝

**Comment by cbernat** [comment]
If there are no further comments @andresmoschini  I will merge this very soon.

**Comment by andresmoschini** [comment]
:shipit: 👏

**Pull request merged**